### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by GregJordanSr**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,42 @@
...
+import Information from "./Components/Information";
+import Video from "./Components/Video";
+
 import "./App.css";
 
-function App() {
+const App = () => {
+  const [state, setState] = useState({
+    
+    date: '',
```
Comment (Zealll) [inline src/App.js:12]: creating these here, actually makes it pointless, cause they get replaced with the `res.data` that is coming from your axios call that you made

File: src/App.js [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
+
   return (
     <div className="App">
-      <p>
-        Read through the instructions in the README.md file to build your NASA
-        app! Have fun 🚀!
-      </p>
+        <Title  title={state.title} />
+        <div className="pic-of-the-day">
+          <Information date={state.date} service={state.service_version} media={state.media_type}  />
```
Comment (Zealll) [inline src/App.js:34]: You don't have to pass these separately, you can actually pass them as `state` and then call them accordingly in your child component

File: src/App.js [hunk 3]
```diff
@@ -1,15 +1,42 @@
...
+        </div>
+        <div>
+          <Video url={state.url} explanation={state.explanation} media={state.media_type} />
+        </div>
     </div>
```
Comment (Zealll) [inline src/App.js]: Same here

File: src/App.js [hunk 4]
```diff
@@ -1,15 +1,42 @@
...
-      <p>
-        Read through the instructions in the README.md file to build your NASA
-        app! Have fun 🚀!
-      </p>
+        <Title  title={state.title} />
+        <div className="pic-of-the-day">
+          <Information date={state.date} service={state.service_version} media={state.media_type}  />
+        </div>
+        <div>
+          <Video url={state.url} explanation={state.explanation} media={state.media_type} />
```
Comment (Zealll) [inline src/App.js:37]: same here

File: src/Components/Information.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+ const Information = props => {
+    return (
+        <div className="info">
+            <p>{props.date}</p>
+            <p>{props.service}</p>
+            <p>{props.media}</p>
```
Comment (Zealll) [inline src/Components/Information.js:8]: So the this actually has another purpose. `media_type` checks whether the `url` is for an image or for a video, so we don't have to display that, but we can use it to conditionally render an image, or a video, depending on the provided url

File: src/Components/Video.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+ const Video = props => {
+    return (
+        <div className="video">
+           {props.media_type === "image" ? <img src={props.url} className=
```
Comment (Zealll) [inline src/Components/Video.js:6]: And this is exactly how we would use it

**Review by Zealll — APPROVED** [review]
Overall good work, but i think there is a bit of a confusion about the hook state, so we will talk about that during our 1on1

**Comment by GregJordanSr** [comment]
Ok

**Pull request merged**